src/store/Rose.java

```diff
@@ -0,0 +1,6 @@
+package store;
+
+public class Rose extends Flowers {
+    public static final int PRICE = 100;
```

Comment:
better dont use constant for it. if you wanted to change price for some kind of rose it would be difficult.
better move this field "price" to the parent class